README.md

```diff
@@ -59,10 +59,14 @@ Partition the input tables into (over decomposition factor) * (number of GPUs) b
 used for computation-communication overlap. This argument has to be an integer >= 1. Higher number
 means smaller batch size. `1` means no overlap. Default: `1`.
 
+**--communicator [STR]**
```

Comment:
We need to update Compilation section or wiki to include details on how to obtain and install NCCL.